packages/patternlab/styleguide/source/_patterns/03-organisms/by-template/page-banner~as-overlay-no-image.json

```diff
@@ -0,0 +1,10 @@
+{
+  "pageBanner": {
+    "size": "overlay",

```

Comment:
Please update packages/patternlab/styleguide/source/_patterns/03-organisms/by-template/page-banner.md to reflect the changes to this component

Reply:
Changes are documented now.